tests/integration/components/isp-consent-form/component-test.js

```diff
@@ -0,0 +1,17 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('isp-consent-form', 'Integration | Component | isp consent form', {
+  integration: true
+});
+
+test('it renders', function(assert) {
+  this.set('studyId', 'test');
+  this.set('grantConsent', (actual) => {
+    assert.deepEqual(actual, true);
+  });
+  // Pass in hasGrantedConsent instead of setting property to override default value
+  this.render(hbs`{{isp-consent-form studyId=studyId grantConsent=(action grantConsent) hasGrantedConsent=true}}`);
+  assert.notOk((this.$('button').attr('disabled')));
+  this.$('button').click();
```

Comment:
So it just tests that the action is called with specific arguments?

Hmm. In this case, I'd say the same advice on `assert.expect` would apply. If you have any thoughts on best practices for making the test really informative, suggestions are welcome!